src/components/Amalgamation/BusinessTable.vue

```diff
@@ -165,6 +167,12 @@ export default class BusinessTable extends Mixins(AmalgamationMixin) {
   }
 
   removeBusiness (index: number): void {
+    // If the company to be deleted is selected as the resulting business name, reset values.
+    const business = this.getAmalgamatingBusinesses.at(index) as any
```

Comment:
Why use `at` instead of `[]` ?

Reply:
No particular reason, I guess it's just muscle memory 😆 

Changed it to `[ ]` Sev.